src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -382,9 +382,9 @@ public void periodic() {
     }
 
     // If the grabber is set to open and it is safe to open, open the grabber. Otherwise, close it.
-    if ((armEncoder.getPosition() > Cal.Lift.GRABBER_CLOSED_ZONE_TOP_DEGREES
-            || armEncoder.getPosition() < Cal.Lift.GRABBER_CLOSED_ZONE_BOTTOM_DEGREES)
-        && grabberOpen) {
+    if (desiredGrabberClosed
+        && (armEncoder.getPosition() > Cal.Lift.GRABBER_CLOSED_ZONE_TOP_DEGREES
+            || armEncoder.getPosition() < Cal.Lift.GRABBER_CLOSED_ZONE_BOTTOM_DEGREES)) {
       grabber.set(Value.kForward);
     } else {
       grabber.set(Value.kReverse);
```

Comment:
This logic isn't right anymore. We should be closed if desired OR if opening is unsafe

Reply:
I've made it so if the variable is false (we want it to be open) and its safe it sets to open.